tools/commentMonitor/README.md

```diff
@@ -5,31 +5,38 @@ Currently it only works with [`issue_comment` event](https://developer.github.co
 
 ### Environment Variables:
 - `GITHUB_TOKEN` : GitHub oauth token used for posting comments and settings the label.
-- Any other environment variable used in any of the comment templates in `eventmap.yml`.
+- Any other environment variable used in any of the comment templates in `config.yml`.
 
 ### Setting up the webhook server
-Running commentMonitor requires the eventmap file which is specified by the `--eventmap` flag.
+To specify the config file for commentMonitor use the the `--config` flag.
 
-The `regex_string`, `event_type` and `comment_template` can be specified in the `eventmap.yml` file.
-
-Example content of the `eventmap.yml` file:
+Example `config.yml` file:
 ```yaml
-- event_type: prombench_stop
-  regex_string: (?mi)^/prombench\s+cancel\s*$
-  comment_template: |
-    Benchmark cancel is in progress.
-  label: prombench
+prefixes:
+  - prefix: /prombench
+    help_template: |
```

Comment:
```suggestion
    help: |
```

Reply:
I was wondering if this change is required because this is also a golang template like `comment_template`? please let me know.